pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -502,6 +502,12 @@ func (p *staticPolicy) takeByTopology(availableCPUs cpuset.CPUSet, numCPUs int)
 		}
 		return takeByTopologyNUMADistributed(p.topology, availableCPUs, numCPUs, cpuGroupSize, cpuSortingStrategy)
 	}
+
+	if p.options.PreferAlignByUnCoreCacheOption {
```

Comment:
Could you add some unit tests around this?

I want to make sure that the fallback (if feature gate is turned off) is covered with some unit tests.

Reply:
Tests are surely welcome, but I think we will need to pass this flag somehow deep down the takeByTopologyNUMAPacked, something like we did for `cpuSortinStrategy` in previous releases.

In any case, I don't think we should proliferate further the `takeByTopology` functions.

Reply:
We have a file for adding tests for policy_static_test. Why can't we at least add some tests based on the right options. 

We should have testing around the case if this feature gate is turned off. We probably won't cover this for e2e tests but we should at least make sure that this feature enablement/disenablement is covered.

Reply:
I fully agree.

Reply:
this seems sufficient for non-regression testing, yes.